src/Stryker.CLI/Stryker.CLI.UnitTest/StrykerCLITests.cs

```diff
@@ -377,301 +124,224 @@ public void StrykerCLI_OnMutationScoreAboveThresholdBreak_ShouldReturnExitCode0(
+        [InlineData("-v", "master")]
+        public void ShouldEnableDiffCompareToDashboardFeatureWhenPassed(params string[] argName)
+        {
+            target.Run(argName);
 
-            target.Run(new string[] { argName });
+            strykerRunnerMock.VerifyAll();
 
-            mock.Verify(x => x.RunMutationTest(It.Is<StrykerOptions>(o => o.BaselineProvider == BaselineProvider.Disk),
-                It.IsAny<IEnumerable<LogMessage>>()));
+            inputs.ProjectVersionInput.SuppliedInput.ShouldBe("master");
         }
 
         [Theory]
-        [InlineData("--baseline-storage-location dashboard")]
-        [InlineData("-bsl dashboard")]
-        public void ShouldSetDashboardBaselineProviderWhenSpecified(string argName)
+        [InlineData("--dashboard-api-key", "1234567890")]
+        public void ShouldSupplyDashboardApiKeyWhenPassed(params string[] argName)
         {
-            var mock = new Mock<IStrykerRunner>(MockBehavior.Strict);
-            var options = new StrykerOptions();
-            var runResults = new StrykerRunResult(options, 0.3);
-
-            mock.Setup(x => x.RunMutationTest(It.IsAny<StrykerOptions>(), It.IsAny<IEnumerable<LogMessage>>())).Returns(runResults);
-
-            var target = new StrykerCLI(mock.Object);
+            target.Run(argName);
 
-            target.Run(new string[] { argName });
+            strykerRunnerMock.VerifyAll();
 
-            mock.Verify(x => x.RunMutationTest(It.Is<StrykerOptions>(o => o.BaselineProvider == BaselineProvider.Dashboard),
-                It.IsAny<IEnumerable<LogMessage>>()));
+            inputs.DashboardApiKeyInput.SuppliedInput.ShouldBe("1234567890");
         }
 
-        [Fact]
-        public void ShouldSetDiskBaselineProviderWhenNotSpecifiedAndNoDashboardReporterSpecified()
+        [Theory]
+        [InlineData("--with-baseline")]
+        public void ShouldSupplyWithBaselineWhenPassed(params string[] argName)
         {
-            var mock = new Mock<IStrykerRunner>(MockBehavior.Strict);
-            var options = new StrykerOptions();
-            var runResults = new StrykerRunResult(options, 0.3);
-
-            mock.Setup(x => x.RunMutationTest(It.IsAny<StrykerOptions>(), It.IsAny<IEnumerable<LogMessage>>())).Returns(runResults);
-
-            var target = new StrykerCLI(mock.Object);
+            target.Run(argName);
 
-            target.Run(new string[] { });
+            strykerRunnerMock.VerifyAll();
 
-            mock.Verify(x => x.RunMutationTest(It.Is<StrykerOptions>(o => o.BaselineProvider == BaselineProvider.Disk),
-                It.IsAny<IEnumerable<LogMessage>>()));
+            inputs.WithBaselineInput.SuppliedInput.Value.ShouldBeTrue();
         }
 
         [Theory]
-        [InlineData("--diff-ignore-files ['**/*.ts']")]
-        [InlineData("-diffignorefiles ['**/*.ts']")]
-        public void ShouldCreateDiffIgnoreGlobFiltersIfSpecified(string argName)
+        [InlineData("--azure-fileshare-sas", "sas")]
```

Comment:
Consider going hardcore, like `--azure-fileshare-shared-access-signature` - otherwise what are the longer option names for? :D

I mean, SAS is [not that obvious abbreviation](https://en.wikipedia.org/wiki/Special_Air_Service).

Reply:
> Consider going hardcore, like --azure-fileshare-shared-access-signature - otherwise what are the longer option names for? :D

I think it's safe to use abbreviations in a context for option names. Otherwise what are our docs for? ;)

Reply:
I think within the context of azure file shares sas is clear enough but I also don't have anything against writing it out.